atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -345,25 +343,26 @@ private TransactionManager serializableInternal(@Output List<AutoCloseable> clos
         Cleaner cleaner = initializeCloseable(() ->
                         new DefaultCleanerBuilder(keyValueService, lockAndTimestampServices.timelock(),
                                         ImmutableList.of(follower), transactionService)
-                                .setBackgroundScrubAggressively(config.backgroundScrubAggressively())
-                                .setBackgroundScrubBatchSize(config.getBackgroundScrubBatchSize())
-                                .setBackgroundScrubFrequencyMillis(config.getBackgroundScrubFrequencyMillis())
-                                .setBackgroundScrubThreads(config.getBackgroundScrubThreads())
-                                .setPunchIntervalMillis(config.getPunchIntervalMillis())
-                                .setTransactionReadTimeout(config.getTransactionReadTimeoutMillis())
-                                .setInitializeAsync(config.initializeAsync())
+                                .setBackgroundScrubAggressively(config().backgroundScrubAggressively())
+                                .setBackgroundScrubBatchSize(config().getBackgroundScrubBatchSize())
+                                .setBackgroundScrubFrequencyMillis(config().getBackgroundScrubFrequencyMillis())
+                                .setBackgroundScrubThreads(config().getBackgroundScrubThreads())
+                                .setPunchIntervalMillis(config().getPunchIntervalMillis())
+                                .setTransactionReadTimeout(config().getTransactionReadTimeoutMillis())
+                                .setInitializeAsync(config().initializeAsync())
                                 .buildCleaner(),
                 closeables);
 
         MultiTableSweepQueueWriter targetedSweep = initializeCloseable(
-                () -> uninitializedTargetedSweeper(metricsManager, config.targetedSweep(), follower,
+                () -> uninitializedTargetedSweeper(metricsManager, config().targetedSweep(), follower,
                         JavaSuppliers.compose(AtlasDbRuntimeConfig::targetedSweep, runtimeConfigSupplier)),
                 closeables);
 
-        Callback<TransactionManager> callbacks = new Callback.CallChain<>(ImmutableList.of(
-                timelockConsistencyCheckCallback(config, runtimeConfigSupplier.get(), lockAndTimestampServices),
+        Callback<TransactionManager> callbacks = new Callback.CallChain<>(
+                timelockConsistencyCheckCallback(config(), runtimeConfigSupplier.get(), lockAndTimestampServices),
                 targetedSweep.singleAttemptCallback(),
-                asyncInitializationCallback()));
+                new DeprecatedTablesCleaner(schemas()),
```

Comment:
Where, if anywhere, did we call `validate` on schemas before? We now additionally call it here, which is likely fine - just trying to understand where we're going to fail if we somehow sneak in an invalid schema.